pyccel/codegen/printing/ccode.py

```diff
@@ -708,13 +766,14 @@ def _print_PythonAbs(self, expr):
         else:
             func = "labs"
         return "{}({})".format(func, self._print(expr.arg))
-
-    def _print_PythonMin(self, expr):
+    
```

Comment:
```suggestion

```

Reply:
@jalalium This change doesn't seem to have gone through right. I wanted to remove your extra whitespace not the whole line :(

Reply:
I fixed it! b631e9f6